packages/app/src/modules/layout.js

```diff
@@ -67,11 +69,14 @@ export const getRetransfer = (layout, transfer, visType) => {
         const destinationAxisId = transfer[id]
         const dimensionsAtDestination = layout[destinationAxisId] || []
 
-        const axisIsFull =
-            dimensionsAtDestination.length ===
-            getAxisMaxNumberOfDimensions(visType, destinationAxisId)
-
-        if (axisIsFull) {
+        if (
```

Comment:
is this if statement the same as `canDimensionBeAddedToAxis`?

Reply:
No.
`canDimensionBeAddedToAxis` will return true if the axis is not full, but in that case we don't need a retransfer.
We only need to work out a retransfer if the axis is full.